packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<nav ref="operatorMenu" :style="style">
```

Comment:
the `nav` may not be necessary, just apply the style to the button and it should work

Reply:
using the nav element for button and menu element to handle the mouse over events